Assets/HoloToolkit-Examples/GazeRuler/Scripts/Test/LineTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary>
+/// it's a test case, we try create line correctly between two points
+/// </summary>
+public class LineTest : MonoBehaviour
+{
+
+
+    public GameObject start;
+    public GameObject end;
+    public GameObject line;
+    public GameObject text;
+    private bool created = false;
```

Comment:
Assets/HoloToolkit-Examples/GazeRuler/Scripts/Test/LineTest.cs(15,18): warning CS0414: The private field `LineTest.created' is assigned but its value is never used